.github/workflows/ci.yml

```
@@ -36,6 +36,7 @@ jobs:
           ERROR_ON_MISSING_EXEC_BIT: true
           VALIDATE_ALL_CODEBASE: true
           VALIDATE_MARKDOWN: false
+          VALIDATE_CHECKOV: false
```

should be true, just disable `CKV2_GHA_1 ` warning
```suggestion
          VALIDATE_CHECKOV: true
```